h/static/scripts/widget-controller.js

```diff
@@ -219,12 +221,10 @@ module.exports = function WidgetController(
     var group = annotationUI.hasSelectedAnnotations() ?
       null : groups.focused().id;
 
-    for (var i=0; i < urls.length; i++) {
-      _loadAnnotationsFor(urls[i], group);
-    }
+    _loadAnnotationsFor(searchUris, group);
```

Comment:
Oh yes, you're right. It did a search first without a `uri` in the params. I moved it down into the same `if` block as the stream filter code.